cellranger/5.0.1/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"
+
+ENV CELLRANGER_MAJ_VER 5.0
+
+RUN apt update && apt install -y curl \
+&& DL_INFO=$(/bin/bash -c "set -o pipefail && curl \"https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/${CELLRANGER_MAJ_VER}/?\" -H 'Connection: keep-alive' \
```

Comment:
so the /bin/bash -c is actually unnecessary here. The command substitution is already acting as its own subshell.

Reply:
The /bin/bash -c is needed because the sub shell is executing as sh which is symlinked to dash and doesn't have pipefail. I found a work around here https://stackoverflow.com/questions/17757039/equivalent-of-pipefail-in-dash-shell but that won't work since it will only fail if the first command fails not any command.

Reply:
Ok. This is good then.